src/pages/Subscriptions/SubscriptionsMonthlyGood.vue

```diff
@@ -146,7 +146,6 @@
 					<subscriptions-monthly-good-cancellation-flow
 						:show-cancel-lightbox="showCancelLightbox"
 						:subscription-id="subscriptionId"
-						v-if="showCancelLightbox"
```

Comment:
This was causing a weird issue where the lightbox 'visible' watcher was not firing. I suppose this v-if takes precedence and the lightbox child component gets removed from the dom before the 'visible' prop is changed